mlrun/data_types/to_pandas.py

```diff
@@ -65,10 +65,10 @@ def toPandas(spark_df):
                 msg = (
                     "toPandas attempted Arrow optimization because "
                     "'spark.sql.execution.arrow.pyspark.enabled' is set to true; however, "
-                    "failed by the reason below:\n  %s\n"
+                    f"failed by the reason below:\n  {str(e)}\n"
```

Comment:
`str()` is redundant here.
```suggestion
                    f"failed by the reason below:\n  {e}\n"
```
This applies to all instances of course.

Reply:
Done in 95cec63.